packages/block-editor/src/components/block-list/block.js

```diff
@@ -578,31 +599,40 @@ function BlockListBlock( {
 					/>
 				) }
 				{ shouldRenderMovers && ( moverDirection === 'vertical' ) && blockMover }
-				{ shouldShowBreadcrumb && (
-					<BlockBreadcrumb
-						clientId={ clientId }
-						ref={ breadcrumb }
-					/>
-				) }
-
 				{ ( isCapturingDescendantToolbars ) && (
 					// A slot made available on all ancestors of the selected Block
 					// to allow child Blocks to render their toolbars into the DOM
 					// of the appropriate parent.
 					<ChildToolbarSlot />
 				) }
-
-				{ ( ! ( hasAncestorCapturingToolbars ) ) && ( shouldShowContextualToolbar || isForcingContextualToolbar.current ) && renderBlockContextualToolbar() }
-
-				{ ( hasAncestorCapturingToolbars ) && ( shouldShowContextualToolbar || isForcingContextualToolbar.current ) && (
-					// If the parent Block is set to consume toolbars of the child Blocks
-					// then render the child Block's toolbar into the Slot provided
-					// by the parent.
-					<ChildToolbar>
-						{ renderBlockContextualToolbar() }
-					</ChildToolbar>
+				{ ( shouldShowBreadcrumb || shouldShowContextualToolbar || isForcingContextualToolbar.current ) && (
+					<Popover
+						noArrow
+						animate={ false }
+						position="top right left"
```

Comment:
`"top right left"` - Is that intentional? What does it mean to be both right and left?

Reply:
The last word, I consider an experimental thing for new, currently the first word is the Y position, the second the direction in which the popover floats (from the arrow), and the last word the corner at which to put the popover. Currently, in `master` we only support positioning it in the middle horizontally. Ideally `Popover` should be refactored to make all these positioning clearer.

Reply:
I added a comment now.

```
// Position above the anchor, pop out towards the right,
// and position in the left corner.
// To do: refactor `Popover` to make this prop clearer.
```